css/index.css

```diff
@@ -0,0 +1,211 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  box-sizing: border-box;
+  padding: 0%;
+  margin: 0;
+}
+/* styling the header */
+.store {
+  color: #e0633a;
+}
+header {
+  background-color: #ebe4e4;
+}
+/* styling h1 */
+h1 {
+  margin-bottom: 50px;
+  color: #de6e49;
+}
+
+/* styling the form */
+form {
+  display: flex;
+  padding-left: 40px;
+  width: 45%;
+  margin: 3% 5%;
+  flex-direction: column;
+  font-size: 15px;
+  height: auto;
+}
+.order-form {
+  display: flex;
+  flex-direction: row;
+}
+
+/* styling the input */
+input,
+#city {
+  padding: 15px;
+  margin-bottom: 20px;
+}
+
+/* styling the label */
+label {
+  margin-bottom: 5px;
+}
+
+/* putting the first and last name in a row */
+.full-name {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* putting first and last name in a column */
+.name-details {
+  display: flex;
+  flex-direction: column;
+  width: 48%;
+}
+
+/* styling address */
+.address-one {
+  display: flex;
+  flex-direction: column;
+}
+.address {
+  display: flex;
+  flex-direction: column;
+  width: 48%;
+}
+
+/* styling city and postcode */
+.city-postcode {
+  display: flex;
+  justify-content: space-between;
+}
+
+#city {
+  border: 0.5px solid black;
+}
+
+/* styling the button */
+button {
+  border-radius: 4px;
+  background-color: #e0633a;
+  cursor: pointer;
+  padding: 15px;
+  width: 120px;
+  color: white;
+  border-style: none;
+  font-size: 11px;
+  margin-top: 25px;
+}
+
+/* styling radio */
+
+.radio-color {
+  display: inline-flex;
+  flex-direction: row;
+  margin-bottom: 30px;
+  justify-items: auto;
+}
+#space-grey {
+  margin-left: 18px;
+}
+/* changing the radio button color to orange when checked */
+
+input[type="radio"] {
+  cursor: pointer;
+  opacity: none;
+  background-color: white;
+  margin: 0;
+  color: grey;
+  width: 15px;
+  height: 15px;
+  border: 1px solid grey;
+  border-radius: 50%;
+  margin-right: 5px;
+  display: grid;
+  place-content: center;
+}
+
+input[type="radio"]::before {
```

Comment:
Have a look at this codepend by Lucy to see an example on how to color radio buttons:
https://codepen.io/LucyMac/pen/VwMZGeg

Reply:
On firefox the radio button is not working but on Chrome it's working. It's great that you put the time to make it work on Chrome. In real work life, the managers want it to work for most browser (it has to be browser compatible.)